test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -169,5 +170,47 @@ private void ValidateOperationOutcome(string actualStatusCode, OperationOutcome
             Assert.Equal(expectedIssueType, issue.Code);
             Assert.Equal(expectedDiagnostics, issue.Diagnostics);
         }
+
+        [Theory]
+        [InlineData("true", "201", true, true)]
+        [InlineData("true", "201", false, true)]
+        [InlineData("true", "201", false, false)]
+        [InlineData("true", "400", true, false)]
+        [InlineData("false", "201", true, false)]
+        [InlineData("false", "201", false, true)]
+        public async Task GivenABundle_WithProfileValidationFlag_ReturnsABundleResponse(string headerValue, string returnValue, bool profileValidation, bool withGender)
```

Comment:
A suggestion as this is a bundle validation, would be to add more than one resource to the bundle.  Perhaps another resource which has no validation errors?  Even a third that has an error?  I wonder how it behaves with multiple resources in the bundle.  Do we see a list of validation errors associated with the different resources?

Reply:
Agreed. I added multiple resources to bundle.
First one is missing gender
Second one is a valid Patient profile.
Third one is missing identifier.